server/capabilities.go

```diff
@@ -142,6 +176,12 @@ func (c *Configuration) EnrichCapabilities(host string) (*paymail.CapabilitiesPa
 	for key, cap := range c.callableCapabilities {
 		payload.Capabilities[key] = serviceUrl + string(cap.Path)
 	}
+	for key, cap := range c.nestedCapabilities {
+		payload.Capabilities[key] = make(map[string]interface{})
+		for nestedKey, nestedCap := range cap {
+			payload.Capabilities[key].(map[string]interface{})[nestedKey] = serviceUrl + nestedCap.Path
```

Comment:
I think it might be safer if we checked the type assertion before we access the nested map.
```golang
nestedObj, ok := payload.Capabilities[key].(map[string]interface{})
if !ok {
   //log warning or even error returning
   continue
}
nestedObj[nestedKey] = serviceUrl + nestedCap.Path
```

Reply:
Added